RevenueCatUI/CustomerCenter/ViewModels/CustomerCenterViewModel.swift

```diff
@@ -165,27 +172,54 @@ private extension CustomerCenterViewModel {
 
     func loadPurchaseInformation(customerInfo: CustomerInfo) async throws {
         let hasActiveProducts =  !customerInfo.activeSubscriptions.isEmpty ||
-        !customerInfo.nonSubscriptions.isEmpty
+            !customerInfo.nonSubscriptions.isEmpty
 
         if !hasActiveProducts {
-            self.purchaseInformation = nil
+            self.purchasesActive = []
             self.state = .success
             return
         }
 
-        guard let activeTransaction = customerInfo.earliestExpiringTransaction() else {
-            Logger.warning(Strings.could_not_find_subscription_information)
-            self.purchaseInformation = nil
-            throw CustomerCenterError.couldNotFindSubscriptionInformation
+        var activePurchases: [PurchaseInformation] = []
+
+        for subscription in customerInfo.subscriptionsByProductIdentifier
```

Comment:
`customerInfo.activeSubscriptions`?

Reply:
`activeSubscriptions` are just the ids 🙃 
`subscriptionsByProductIdentifier` has the actual subscription info